doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -382,6 +382,9 @@ nRF Desktop
     This ensures correct behavior if another source requests Constant Latency sub-power mode through the nrfx API.
   * The :ref:`CONFIG_DESKTOP_CONSTLAT_DISABLE_ON_STANDBY <config_desktop_app_options>` Kconfig option depends on :kconfig:option:`CONFIG_CAF_PM_EVENTS`.
     CAF Power Management events support is necessary to disable constant latency interrupts on standby.
+  * The ``APPLICATION_CONFIG_DIR`` CMake option is set in application's :file:`CMakeLists.txt` file.
+    The CMake option was previously set through sysbuild's file:`CMakeLists.txt` file.
+    Change ensures proper behavior in case application source directory name is changed.
```

Comment:
```suggestion
    Previously, the option was set through sysbuild's :file:`CMakeLists.txt` file.
    This change ensures proper behavior in case the source directory name of the application is changed.
```